etc/kayobe/kolla.yml

```
@@ -307,6 +307,10 @@ kolla_build_blocks:
     RUN sed -i 's/"pc-q35-rhel8.5.0"/"pc-q35-*"/' /usr/share/qemu/firmware/50-edk2-ovmf-cc.json
     {% endif %}
     {% endraw %}
+  neutron_server_footer: |
+    # This is a workaround for a bug in etcd3gw that was fixed in 1.0.1, but not in v0.2.3.
+    # https://opendev.org/openstack/etcd3gw/commit/ed899b34e464862525f76fff2377a2cceeb65af7
+    RUN sed -i 's/decode_unicode=True/decode_unicode=False/' /var/lib/kolla/venv/lib/python3.8/site-packages/etcd3gw/watch.py
```

```suggestion
    RUN sed -i 's/decode_unicode=True/decode_unicode=False/' /var/lib/kolla/venv/lib/python3.6/site-packages/etcd3gw/watch.py
```
Issue with python version showed up in the builds:
```
ERROR:kolla.common.utils.neutron-server:Error'd with the following message\nERROR:kolla.common.utils.neutron-server:The command '/bin/sh -c sed -i 's/decode_unicode=True/decode_unicode=False/' /var/lib/kolla/venv/lib/python3.8/site-packages/etcd3gw/watch.py' returned a non-zero code:
```